frontend/.eslintrc

```diff
@@ -32,7 +32,7 @@
     "sourceType": "module"
   },
   // Ignore our auto-generated code
-  "ignorePatterns": ["src/autogen/*"],
+  "ignorePatterns": ["src/autogen/*", "**/vendor/*"],
```

Comment:
Nit: the comment above should now read "// Ignore our auto-generated and vendored code"